app/controllers/checkouts_controller.rb

```diff
@@ -7,4 +7,12 @@ def create
 
     redirect_to root_path
   end
+
+  def destroy
+    @book = Book.find(params[:book_id])
+    @checkout = Checkout.where(user: current_user, book: @book)
+    @checkout.destroy_all
```

Comment:
@derekprior, Thanks for this great explanation. You are right; you don't need to identify yourself. At a library, I just place my books in the return trolley and the unique record of that checkout it destroyed/archived once an admin scans it as a returned item. 

I like options 2 and 3. I admit 2 is slightly more familiar to me, but I am definitely open to trying option 3 as I have  less experience dealing with plain Ruby objects. 

I also agree with your comment down below. This UI serves as a way to get things working quickly. A dedicated view where users see the books they have checked out (and return them) is much better, and what I planned to build next. That way the list of unavailable books on the books `index` page does not have to worry about who can return which books. 

Is the best course of action to change this PR so that it uses a user profile page (or something similar) to handle book returns and viewing currently checked out books? That is what I lean.